pytorch_lightning/trainer/trainer.py

```diff
@@ -532,6 +542,7 @@ def get_init_arguments_and_types(cls) -> List[Tuple[str, Tuple, Any]]:
              ('max_epochs', (<class 'int'>,), 1000),
              ...
              ('precision', (<class 'int'>,), 32),
+             ('prepare_data_per_node', (<class 'bool'>,), True),
```

Comment:
it is in the wrong position. it should go after auto_scale_batch, which is not listed here. so the solution is to remove it.
also you removed 
doctest: +ELLIPSIS +NORMALIZE_WHITESPACE
which I am fairly confident is needed

Reply:
what is it here?
if i remove it, it still fails

Reply:
well maybe I was wrong and it is just  the missing +ELLIPSIS +NORMALIZE_WHITESPACE
cloned and doctests run locally.
so i think its ok now